app/assets/javascripts/short-form/templates/d6-status-vouchers.html.slim

```diff
@@ -8,7 +8,7 @@ form novalidate="" name="form.applicationForm" ng-submit="submitForm({path: 'dah
     .radio-group.padding-top--half role="radiogroup" aria-label="{{'D6_STATUS_VOUCHERS.TITLE' | translate}}"
       radio-block-item value='Yes' user='application' name='householdVouchersSubsidies' ng-required='true'
         | {{'T.YES' | translate}}
-      radio-block-item value='No' user='application' name='householdVouchersSubsidies' ng-required='true'
+      radio-block-item value='No' user='application' name='householdVouchersSubsidies' ng-required='true' ng-click="invalidateIncomeForm()"
```

Comment:
@chrisdolendo exactly, it makes it so that if the income form is basically marked as "incomplete" so you have to complete it again -- because if you marked "Yes" then you would have skipped the income check. So we need to make sure you run through the income check this time.